example/main.py

```diff
@@ -1,19 +1,26 @@
+from enum import Enum
+from ipaddress import IPv4Address, IPv6Address
 from typing import Annotated, List, Literal, Union
 
 from fastapi import FastAPI
 from fastapi.exceptions import RequestValidationError
 from fastapi.responses import JSONResponse
-from pydantic import BaseModel, Field
+from pydantic import BaseModel, Field, field_validator, ValidationError
 from pydantic.networks import AnyHttpUrl
 from starlette.requests import Request
 
 from fastapi_validation_i18n import i18n_exception_handler, I18nMiddleware, Translator
 
-app = FastAPI()
+app = FastAPI(debug=True)
 
 app.add_middleware(I18nMiddleware, locale_path='example/locale')
```

Comment:
**Suggestion:** Use f-strings for string concatenation for better readability and performance.
```suggestion
app.add_middleware(I18nMiddleware, locale_path=f"{Path.cwd()}/example/locale")
```